src/java/org/jivesoftware/openfire/session/OutgoingServerSession.java

```diff
@@ -79,4 +79,6 @@
      * @return True if the subdomain was authenticated by the remote server.
      */
     boolean authenticateSubdomain(String domain, String hostname);
+

```

Comment:
Tsk. All other definitions in this interface definition got nice documentation. :)

Reply:
I have written nice documentation now.